sjb/config/test_cases/ami_build_origin_int_rhel_base.yml

```diff
@@ -28,6 +28,10 @@ actions:
     script: |-
       oct prepare repositories
   - type: "script"
+    title: "install mvn and java"
+    script: |-
+      sudo yum install -y maven java-1.8.0-openjdk-devel
```

Comment:
With the install of the jdk, what fills in the `?` noted at https://github.com/openshift/origin-ci-tool/blob/master/oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml#L39 ?